web/templates/blocks/clusters_table.html.tmpl

```diff
@@ -4,26 +4,36 @@
             <thead>
             <tr>
                 <th scope='col'></th>
-                <th scope='col'>SID</th>
                 <th scope='col'>Cluster name</th>
                 <th scope='col'>Cluster Id</th>
                 <th scope='col'>Cluster type</th>
+                <th scope='col'>SID</th>
                 <th scope='col'>Nr. Hosts</th>
                 <th scope='col'>Nr. Resources</th>
+                <th scope='col'>Tags</th>
             </tr>
             </thead>
             <tbody>
             {{- range . }}
-                <tr class='clickable' onclick="window.location='/clusters/{{ .Id }}'">
+                <tr>
                     <td class="row-status">{{ template "health_icon" .Health }}</td>
-                    <td>{{- range $i, $v := .SIDs }}{{- if $i }},{{- end }} {{ . }}{{- end }}</td>
                     <td>
-                        {{- if .HasDuplicatedName }}<i class="eos-icons eos-18 text-warning" data-toggle="tooltip" data-original-title="This cluster has a duplicated name">warning</i>{{- end }}{{ .Name }}
+                        {{- if .HasDuplicatedName }}<i class="eos-icons eos-18 text-warning" data-toggle="tooltip" data-original-title="This cluster has a duplicated name">warning</i>{{- end }}<a href="/clusters/{{ .Id }}">{{ .Name }}</a>
                     </td>
-                    <td>{{ .Id }}</td>
+                    <td><a href="/clusters/{{ .Id }}">{{ .Id }}</a></td>
                     <td>{{ .Type }}</td>
+                    <td>{{- range $i, $v := .SIDs }}{{- if $i }},{{- end }} {{ . }}{{- end }}</td>
                     <td>{{ .HostsNumber }}</td>
                     <td>{{ .ResourcesNumber }}</td>
+                    <td>
```

Comment:
This looks like something that could go in other template, passing the maybe data-url as param.
I guess you already checked and has some drawback hehe

Reply:
yes i tried but since there is no way that i know to pass variables directly to a nested template, i decided to repeat it, since i did not want to pass them from the backend.